main.js

```diff
@@ -0,0 +1,147 @@
+const VALIDATE = (prop) => {
+    if (prop === undefined || prop === null) {
+        return true;
+    }
+    return false;
+}
+
+// 1. Cтворити функцію, яка створює та реєструє нову команду у автопробізі. Під час виклику функції потрібно передати дані про команду - назва команди, ім'я водія, рік народження водія, марка автомобіля, спонсор команди та чи водій допущений до автопробігу
+
+// 1st var
+const CAR_RACE_TEAM = (teamName, driverName, yearOfBirth, carBrand, sponsor, isAllowed) => {
+    if (VALIDATE(teamName) || VALIDATE(driverName) || VALIDATE(yearOfBirth) || VALIDATE(carBrand) || VALIDATE(sponsor) || VALIDATE(isAllowed) || isNaN(yearOfBirth)) {
+        console.log("Something went wrong! Check the information and try to fill one more time.");
+    } else {
+        const NEW_TEAM = {
+            teamName: teamName,
+            driverName: driverName,
+            yearOfBirth: yearOfBirth,
+            carBrand: carBrand,
+            sponsor: sponsor,
+            isAllowed: isAllowed,
+        } 
+        console.log(NEW_TEAM);
+    }
+}
+CAR_RACE_TEAM('Speed', 'Antony', 1992, 'Ferrari', 'F1', true)
+
+// 2nd var
+function NEW_TEAM (teamName, driverName, yearOfBirth, carBrand, sponsor, isAllowed) {
+    if (VALIDATE(teamName) || VALIDATE(driverName) || VALIDATE(yearOfBirth) || VALIDATE(carBrand) || VALIDATE(sponsor) || VALIDATE(isAllowed) || isNaN(yearOfBirth)) {
+        console.log("Something went wrong! Check the information and try to fill one more time.");
+    } else {
+    this.teamName = teamName;
+    this.driverName = driverName;
+    this.yearOfBirth = yearOfBirth;
+    this.carBrand = carBrand;
+    this.sponsor = sponsor;
+    this.isAllowed = isAllowed;
+    }
+}
+
+const NEW_TEAM_1 = new NEW_TEAM ('Speed', 'Antony', 1992, 'Ferrari', 'F1', ); 
+console.log(NEW_TEAM_1); // Something went wrong! Check the information and try to fill one more time.
+const NEW_TEAM_2 = new NEW_TEAM ('Fire', 'John', 1990, 'Lamborghini', 'F1', false);
+console.log(NEW_TEAM_2);
+
+// 2. Запитати дані (через prompt та циклічно) про нового користувача у системі та зберегти його в об'єкт. Дані про користувача - логін, пароль, місто, країна, стать, вік
+const USER_1 = {};
+USER_1.login = prompt("Enter login");
+USER_1.pass = prompt("Enter password");
+USER_1.city = prompt("What is your city?");
+USER_1.country = prompt("What country are you from?");
+USER_1.male = prompt("What is your mail?");
+USER_1.age = prompt("What is your age?");
+console.log(USER_1)
+
+
+// 3. Створити функцію, яка буде міняти дані в конкретного користувача створеного пунктом вище. Наприклад сhangeUserData(user_1, city, 'Kherson). Де user_1 - обʼєкт в якому буде мінятись, city - поле, яке буде мінятися на нове значення - 'Kherson'
+const CHANGE_USER_DATA = (obj) => {
```

Comment:
According to the task, you should pass two params in the function, the object that will change and the exact field to change, and the value
It should be something like
```suggestion
const CHANGE_USER_DATA = (obj, field, value) => {
return obj.field = value
```

Also nice to have validation, to verify if the field exists and passing data at least not empty